index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="03/23/2023 - This is a Catholic Music Concert to be ready for 2023 in Rome city of Italy in July month.">
+  <meta name="keyword" content="Catholic, Music, Meet, Cristian">
+  <title>Catholic Music Concert - United by the same Faith</title>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3848942bcd.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+
+<body>  
+  <header>
+    <div class="mylinks">
+      <ul class="link-menu">
+        <li class="link-item"><a ><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li class="link-item"><a ><i class="fa-brands fa-twitter"></i></a></li>
+        <li class="link-item"><a >English</a></li>
+        <li class="link-item"><a >My Page</a></li>
+        <li class="link-item"><a >Logout</a></li>
+      </ul>
+    </div>
+    <div class="menu-cont">
+      <i class="fa-sharp fa-solid fa-bars"></i>
+      <a href="index.html"><img src="images/logo.png" alt="Logo Concert Music Catholic" class="logo"></a>
+      <nav class="nav-menu">
+        <ul class="menu-dk">
+          <li><a class="menu-link" href="about.html">About</a></li>
+          <li><a class="menu-link" href="#program">Program</a></li>
+          <li><a class="menu-link" href="#join">Join</a></li>
+          <li><a class="menu-link" href="#sponsors">Sponsor</a></li>
+          <li><a class="menu-link" href="#news">News</a></li>
+        </ul>
+        <button class="btn-campain" type="button"><a href="#">CMC World</a></button>
+      </nav>
+    </div>      
+  </header>  
+  <div class="page">
+    <main>
+      <section class="principal">
+      <div class="tittle">
+        <h2 class="txt-tittle">"United by the same Faith"</h2>
+        <h1 >CATHOLIC MUSIC CONCERT<br>WORLD TOUR July 2023</h1>
+        <p class="description">In july, a series of presentations will be held in Rome, Italy, to praise and give glory to our Lord Jesus Christ, accompanied by the best representatives of catholic music.</p>
+        <div class="date-site">
+          <p class="date">2023.07.20(THU) ~ 22(SAT)</p>
+          <p class="site">@Auditorio Parco della Musica - Roma RM, Italia</p>
+        </div>        
+      </div>
+      </section>
```

Comment:
your home page has the following issues that require immediate attention

- your nav menu has an overflow which is making you webpage look highly unprofessional as seen in the image below 👍🏾 
- I'll highly recommend that you nav bar border's font should be reduced or removed completely as one of the requirements is to make your design almost identical to the given design as seen below
- your `principal` section width should be 100% taking up the full width of the screen.

your design
![image](https://user-images.githubusercontent.com/60612329/227381111-8e7b5028-844e-4f24-9408-69e772c9487c.png)

template design
![image](https://user-images.githubusercontent.com/60612329/227381229-2c01c3e3-b046-409d-b770-5db4261bbf99.png)

Reply:
@alexander16108, I correct:

![image](https://user-images.githubusercontent.com/110146573/227424799-a3c2da7b-6d21-4cd5-a35f-12bcf2d61fb9.png)